### ISSUE TITLE
Time Range - unrecognized arguments on Mac Terminal

### ISSUE THREAD

**Issue by mustafa3000** [hardikvasa/google-images-download#105]
Hi!

First off, thanks for creating this - what an amazing resource! I'm having trouble executing a time_range argument when running command line terminals on Mac. I'm a complete newbie, so I may not be typing the command for time range correctly (have tried lots of variations with & without brackets, etc). Any guidance is appreciated!

I keep trying to run the below but get "unrecognized arguments" errors no matter what variation of the arguments i use:
`googleimagesdownload --keywords "economy chart" --limit 10  --time_range {"time_min":"01/01/2018","time_max":"04/01/2018"}

#result returned below
googleimagesdownload: error: unrecognized arguments: time_max:04/01/2018`

I've also received an error when using '-wr' in place of '--time_range'
`googleimagesdownload --keywords "economy chart" --limit 10  --wr {"time_min":"01/01/2018","time_max":"04/01/2018"}

#result returned below
googleimagesdownload: error: argument -w/--time: invalid choice: 'time_min:01/01/2018' (choose from 'past-24-hours', 'past-7-days')`

Thanks!

**Comment by hardikvasa** [comment]
@mustafa3000 could you resolve your issue?

**Comment by mustafa3000** [comment]
Hi @hardikvasa - onlyl partly :) I figured out how to use the command - BUT the "time range" filter doesn't seem to work (on Mac terminal). 
rather when i compare the downloaded results against searching on google images, i get very different results when using "time range" argument. 

For example:
when using a command without the time range filter like
`googleimagesdownload --keywords "daily chart " --limit 10 --time 'past-7-days'`
i will compare the results i download against a live google image search, and will see the SAME results.

However, when i add the time range argument, 
`googleimagesdownload --keywords "daily chart " --limit 10 --time_range '{"time_min":"01/01/2017","time_max":"05/01/2018"}'`

i do NOT get the same results. it seems the time range throws the search off and not sure if anyone else has this issue. Any guidance is helpful.
Thanks!

**Comment by ty-vincenthung** [comment]
Hi hardikvasa and nustafa3000,
I am glad to be here and I found that in source code the argument making for time_range is incorrect and I corrected it in my side but I am not sure how to push a new branch/commit to your project.

The issue is in around line 365, 
```
if arguments['time_range']:
            json_acceptable_string = arguments['time_range'].replace("'", "\"")
            d = json.loads(json_acceptable_string)
            time_range = ',cdr:1,cd_min:' + d['time_min'] + ',cd_max:' + d['time_max']
```

after changing the d['time_max'], it should be working fine. :)

**Comment by hardikvasa** [comment]
thanks @mustafa3000 and @ty-vincenthung for the inputs. The changes were merged in #152 so this should be fixed now.
